### PR TITLE
Update config guide with more observations

### PR DESCRIPTION
**Pull request by dafeda**
- [ ] PR title captures the intent of the changes, and is fitting for release notes.
- [ ] Added appropriate release note label
- [ ] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [ ] Make sure tests pass locally (after every commit!)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

File: docs/getting_started/configuration/poly_new/guide.rst [hunk 1]
```diff
@@ -445,25 +449,47 @@ With the final configuration:
...
 .. image:: with_observations/coeff_a.png
 
 .. image:: with_observations/coeff_b.png
 
 .. image:: with_observations/coeff_c.png
 
+Adding Observations
+*******************
+Notice that there is not much improvement in the estimate of parameter ``b``.
+Let's try adding more obeservations with lower uncertainty and run a new experiment.
```
Comment (oyvindeide) [inline docs/getting_started/configuration/poly_new/guide.rst:474]:
```suggestion
Let's try adding more observations with lower uncertainty and run a new experiment.
```

File: docs/getting_started/configuration/poly_new/guide.rst [hunk 2]
```diff
@@ -445,25 +449,47 @@ With the final configuration:
...
+
+Modify ``poly_eval.py`` to generate 50 responses:
+
+.. literalinclude:: with_more_observations/poly_eval.py
+
+Remove index list from ``observations`` as we now use all 50 observations:
+
+.. literalinclude:: with_more_observations/observations
+
+Re-run ``Ensemble smoother`` and notice that the estimate of ``b`` has improved:
```
Comment (oyvindeide) [inline docs/getting_started/configuration/poly_new/guide.rst:489]: Perhaps add a bit more explaining why that improves the result?

**Review by oyvindeide — APPROVED** [review]
LGTM! Some minor comments

File: docs/getting_started/configuration/poly_new/guide.rst (hunk 2 above)
Comment (dafeda) [reply docs/getting_started/configuration/poly_new/guide.rst:489]: I'll add more when inspiration strikes.

**Pull request merged**